accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -500,16 +512,19 @@ export class OptionsApp extends React.Component<{}, OptionsAppState> {
 
                         {typeof document === 'undefined' ? null : ReactDOM.createPortal(<Modal
                             aria-label="Guidelines information"
-                            modalHeading="Selecting accessibility guidelines"
+                            modalHeading="Selecting an accessibility guideline"
                             passiveModal={true}
                             open={this.state.modalGuidelines}
                             onRequestClose={(() => {
                                 this.setState({ modalGuidelines: false });
                             }).bind(this)}
                         >
-                            <p style={{ maxWidth: "100%" }}><strong>IBM Accessibility: </strong> Rules for WCAG 2.1 AA plus additional IBM requirements</p>
-                            <p style={{ maxWidth: "100%" }}><strong>WCAG 2.1 (A, AA): </strong> This is the current W3C recommendation. Content that conforms to WCAG 2.1 also conforms to WCAG 2.0</p>
-                            <p style={{ maxWidth: "100%" }}><strong>WCAG 2.0 (A, AA): </strong> Referenced by US Section 508, but not the latest W3C recommendation</p>
+                            <p style={{ maxWidth: "100%" }}><strong>IBM Accessibility 7.2</strong>: Rules for WCAG 2.1 plus additional IBM requirements
+                            <p style={{ maxWidth: "100%" }}><strong>WCAG 2.2 (A, AA)</strong>: Rules for the latest W3C specification. The "Preview Rules" rule set must also be selected.</p>
```

Comment:
```suggestion
                            <p style={{ maxWidth: "100%" }}><strong>WCAG 2.2 (A, AA)</strong>: Rules for the latest W3C specification. Content that conforms to WCAG 2.2 also conforms to 2.1 and 2.0.</p>
```
Use does not need to use the "preview" ruleset to get WCAG 2.2